proxysql-admin

```diff
@@ -1507,7 +1507,20 @@ function enable_proxysql() {
                            "\n-- Please check the ProxySQL connection parameters and status."
     fi
   fi
+  
+  # Update mysql server version details in proxysql db 
+  proxysql_mysql_version_string=$(proxysql_exec "$LINENO" "select variable_value from global_variables where variable_name like 'mysql-server_version'")
```

Comment:
Every call to proxysql_exec and mysql_exec needs to be checked for an error return.